go/store/hash/base32.go

```diff
@@ -25,11 +25,13 @@ import "encoding/base32"
 
 var encoding = base32.NewEncoding("0123456789abcdefghijklmnopqrstuv")
 
-func encode(data []byte) string {
+// Encode returns the base32 encoding in the Dolt alphabet.
+func Encode(data []byte) string {
 	return encoding.EncodeToString(data)
 }
 
-func decode(s string) []byte {
+// Decode returns the bytes represented by the Base32 string using the Dolt alphabet.
+func Decode(s string) []byte {
```

Comment:
This package already public exposes `MaybeParse` and `Parse`, and it exposes `(h Hash) String() string`.

Are we sure we want to expose these functions? Is `Decode` actually safe to use anywhere / do we actually want it used?

Reply:
Decode is used once by the `padStringAndDecode` method. Honestly what it's used for looks like it could be converted into a constants - upper and lower bounds of an address. I'll fix this up.

Reply:
hmmmm... it's used by some go tests too.